controlpanel/frontend/jinja2/datasource-detail.html

```diff
@@ -113,46 +76,11 @@ <h2 class="govuk-heading-m">Data access group</h2>
     </tfoot>
   </table>
 
-  {% if request.user.has_perm('api.create_users3bucket', bucket) %}
-  {% if users_options|length %}
-    <form action="{{ url('grant-datasource-access', kwargs={"pk": bucket.id}) }}" method="post">
-    {{ csrf_input }}
-    <div class="govuk-form-group">
-      <label class="govuk-label" for="user_id">Grant access to this data to other users</label>
-      <select class="govuk-form-control no-blank autocomplete-select" id="user_id" name="user_id">
-        <option value="">Select a user</option>
-        {% for user in users_options %}
-        <option value="{{ user.auth0_id }}">{{ user_name(user) }}</option>
-        {% endfor %}
-      </select>
-    </div>
-
-    <div class="govuk-form-group panel panel-border-narrow js-hidden" id="data-access-level-panel">
-      {{ govukRadios({
-        "name": "access_level",
-        "fieldset": {
-          "legend": {
-            "text": "Data access level",
-          },
-        },
-        "hint": {
-          "html": "<strong>NOTE:</strong> Making a user an admin will allow them to confer access rights on additional users.",
-        },
-        "items": [
-          {"value": "readonly", "text": "Read only", "checked": True},
-          {"value": "readwrite", "text": "Read/write"},
-          {"value": "admin", "text": "Admin"},
-        ]
-      }) }}
-    </div>
-
-    <div class="govuk-form-group">
-      <button class="govuk-button">Grant access</button>
-    </div>
-  </form>
-  {% else %}
-  <p class="govuk-body">(All available users already have access to this data source.)</p>
-  {% endif %}
+  {% if request.user.has_perm('api.create_users3bucket', bucket) and users_options|length %}
+    <a href="{{ url('grant-datasource-access', kwargs={'pk': bucket.pk}) }}"
```

Comment:
Wasn't this a `POST` request? Wouldn't this make it a `GET` request?

Reply:
Instead of submitting an inline form (which was indeed a POST request), this link takes you to a separate form page